src/main/java/com/ldtteam/structurize/event/EventSubscriber.java

```diff
@@ -83,4 +97,103 @@ public static void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
         }
         Manager.onWorldTick((ServerWorld) event.world);
     }
+
+
+    @SubscribeEvent
+    public static void onMissingBlockMappings(final RegistryEvent.MissingMappings<Block> missingBlockEvent)
+    {
+        final ImmutableList<RegistryEvent.MissingMappings.Mapping<Block>> missingBlocks = missingBlockEvent.getMappings(Constants.MOD_ID);
+        if (missingBlocks.isEmpty())
+        {
+            return;
+        }
+
+        handleMissingMappingsDueToRestructureIn9af7543d7dcefa0bf7b52c2e809d6c8d48b803a6(missingBlocks);
```

Comment:
this name should be rather in jdoc (or at least not contain the entire commit hash)

Reply:
I don't want them to be in the jdoc.
I made the decision that i wanted in the name because there might be several remaps that we need in the future.